cloudformation/cw-dashboard-substack.cfn.yaml

```diff
@@ -0,0 +1,241 @@
+Conditions:
+  CreateEFSSubstack: !Equals
+    - !Ref 'CreateEFSSubstack'
+    - 'true'
+  CreateFSXSubstack: !Equals
+    - !Ref 'CreateFSXSubstack'
+    - 'true'
+  CreateRAIDSubstack: !Equals
+    - !Ref 'CreateRAIDSubstack'
+    - 'true'
+  # Duplicate of the raid-substack code
+  UseRaidVol1: !Not
+    - !Equals
+      - !Select
+        - '0'
+        - !Ref 'RAIDOptions'
+      - NONE
+  UseRaidVol2: !And
+    - !Not
+      - !Equals
+        - !Select
+          - '2'
+          - !Ref 'RAIDOptions'
+        - '1'
+    - !Condition 'UseRaidVol1'
+  UseRaidVol3: !And
+    - !Not
+      - !Equals
+        - !Select
+          - '2'
+          - !Ref 'RAIDOptions'
+        - '2'
+    - !Condition 'UseRaidVol2'
+  UseRaidVol4: !And
+    - !Not
+      - !Equals
+        - !Select
+          - '2'
+          - !Ref 'RAIDOptions'
+        - '3'
+    - !Condition 'UseRaidVol3'
+  UseRaidVol5: !And
+    - !Not
+      - !Equals
+        - !Select
+          - '2'
+          - !Ref 'RAIDOptions'
+        - '4'
+    - !Condition 'UseRaidVol4'
+  # Duplicate of the ebs-substack code
+  UseEBSVol2: !Not
+    - !Equals
+      - !Ref 'NumberOfEBSVol'
+      - '1'
+  UseEBSVol3: !And
+    - !Not
+      - !Equals
+        - !Ref 'NumberOfEBSVol'
+        - '2'
+    - !Condition 'UseEBSVol2'
+  UseEBSVol4: !And
+    - !Not
+      - !Equals
+        - !Ref 'NumberOfEBSVol'
+        - '3'
+    - !Condition 'UseEBSVol3'
+  UseEBSVol5: !And
+    - !Not
+      - !Equals
+        - !Ref 'NumberOfEBSVol'
+        - '4'
+    - !Condition 'UseEBSVol4'
+Outputs: {}
+Parameters:
```

Comment:
Let's put the Parameters as first item, then Conditions, then Resources and Outputs at the end.

Reply:
I am doing that because the order in the other EFS,EBS,FSx,RAID substacks is Conditions, Outputs, Parameters and then Resources (COPR). Batch and aws-parallelcluster (excluding Mappings for this one) are PCRO and CloudWatchLogs is CPRO. Shouldn't we follow the same pattern every time?

Reply:
Let's use the logical order suggested by official [CFN guide](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/template-anatomy.html):
> Some sections in a template can be in any order. However, as you build your template, it can be helpful to use the logical order shown in the following list because values in one section might refer to values from a previous section.

In our case is PCRO, as suggested before.

Reply:
Done

Reply:
Now it is really done, had forgotten to do it in aws-parallelcluster.cfn.json.